io_scene_godot/export_godot.py

```diff
@@ -243,18 +374,20 @@ def load_supported_features(self):
                         continue
 
                     _, version_str = tuple(line.split("="))
-                    conf_versiton = int(version_str)
+                    conf_version = int(version_str)
                     break
 
-        if conf_versiton < 2:
+        if conf_version < 2:
             logging.error(
                 "Godot version smaller than 3.0, not supported by this addon")
 
-        if conf_versiton >= 4:
+        if conf_version >= 4:
             # godot >=3.1
             self.config["feature_bezier_track"] = True
 
     def export(self):
+        # pylint: disable-msg=too-many-branches
```

Comment:
Generally it's better to fix this issue rather than disable the warnings. It's like turning off the fire alarm rather than putting out the fire.
Is there some way this function can be re-structured to use fewer `if` statements and generally be shorter? 

What about:
 - Shifting the prefix detection to a new function
 - Moving the folder creation to a new function.

Reply:
I probably would have done so If it wasn't already done in the export_scene function.
But I will try to implement your suggestions.

I also found a bug but I am not sure If it is related to my code, I will do further investigation there.